src/nvme/mi.c

```diff
@@ -539,7 +554,8 @@ static void nvme_mi_control_init_req(struct nvme_mi_req *req,
 
 	control_req->hdr.type = NVME_MI_MSGTYPE_NVME;
 	control_req->hdr.nmp = (NVME_MI_ROR_REQ << 7) |
-		(NVME_MI_MT_CONTROL << 3); /* we always use command slot 0 */
+		(NVME_MI_MT_CONTROL << 3)|
```

Comment:
minor: space around the `|` operator, to be consistent with the line above

(and elsewhere in the change)